config/manager/kustomization.yaml

```diff
@@ -1,10 +1,16 @@
 resources:
-- manager.yaml
+  - manager.yaml
```

Comment:
should we remove the first indent to make the file consistent with the other operators ?